source/sass/buyers-guide/views/product.scss

```diff
@@ -89,13 +92,32 @@ $product-detail-padding-x: (
     }
   }
 
+  h3.body,
   h4 {
     font-family: "Nunito Sans";
   }
 
-  .creep-face-emoji {
-    top: 3px;
-    margin: 0 0.25em 0 0;
+  .worst-case-box {
+    position: relative;
+    border-width: 1px;
+    border-image: linear-gradient(to left, $pni-pink 0%, $pni-blue 100%) 1 / auto;
```

Comment:
it looks like Chrome doesn't like this, but if I add the `-webkit-` prefix it looks like Chrome forgets what a border is, and just makes it the entire background O_O

![image](https://user-images.githubusercontent.com/177243/67021639-c9af5000-f0b4-11e9-8e15-abdd6fd8b1d1.png)

Reply:
The rule specified in https://css-tricks.com/gradient-borders-in-css/ does seem to work in both Chrome and Firefox

Reply:
Thanks for the link @Pomax . It turned out just adding `border-style: solid` fixed the Chrome issue. Phew! One fewer divs to maintain 😂